api/types/event/snapshot.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package containerd.v1.types;
```

Comment:
Doesn't matter to me.  I kept them in the `events` type package to make it logical for events but I could see it in the others too.  I don't have a strong feeling either way.